src/index.js

```diff
@@ -107,11 +136,17 @@ worksheet.getRow(1).font = {
 
 worksheet.addRow();
 
-const legendRowGood = worksheet.addRow(['', '=', 'Correctly marked as "Must Support"']);
-const legendRowBad = worksheet.addRow(['', '=', 'Not marked as "Must Support"']);
+const legendRowGood = worksheet.addRow(['', '=', 'Correctly marked as "Must Support" in QI-Core']);
+const legendRowBad = worksheet.addRow(['', '=', 'Not marked as "Must Support" in QI-Core']);
+const legendRowWarn = worksheet.addRow([
+  '',
+  '=',
+  'Marked as "Must Support" in QI-Core but not in US-Core (added data element)'
+]);
 
 legendRowGood.getCell(1).fill = GOOD_FILL_CONFIG;
 legendRowBad.getCell(1).fill = BAD_FILL_CONFIG;
+legendRowWarn.getCell(1).fill = WARN_FILL_CONFIG;
 
```

Comment:
Not sure how much of a pain this would be to add code for, but when I open up the `output.xlsx` file, some of the cells are too small (ex. `AllergyIntolerance` and `Observation` get cut off in the first row). It would be nice to dynamically change the size of the cells depending on the text that's present, but not super necessary

Reply:
I'll give this a quick investigation and see if I find anything easy. If not, we can pawn this off to a new task

Reply:
Pushed up a change to make the column width dynamic such that it fits the longest string in the column (I ignored the legend at the bottom)